server/parsec/components/postgresql/organization_update.py

```diff
@@ -83,12 +94,20 @@ async def organization_update(
         fields["user_profile_outsider_allowed"] = user_profile_outsider_allowed
     if with_minimum_archiving_period:
         fields["minimum_archiving_period"] = minimum_archiving_period
+    if with_tos:
+        if tos is None:
+            fields["tos_updated_on"] = None
+            fields["tos_url"] = None
+        else:
+            fields["tos_updated_on"] = now
+            fields["tos_url"] = tos
```

Comment:
`tos_per_locale_urls`?
```suggestion
            fields["tos_updated_on"] = None
            fields["tos_per_locale_urls"] = None
        else:
            fields["tos_updated_on"] = now
            fields["tos_per_locale_urls"] = tos
```

Reply:
done.